src/qt/coincontroldialog.cpp

```diff
@@ -392,10 +388,9 @@ void CoinControlDialog::viewItemChanged(QTreeWidgetItem* item, int column)
         if (ui->treeWidget->isEnabled()) // do not update on every click for (un)select all
             CoinControlDialog::updateLabels(model, this);
     }
-
     // TODO: Remove this temporary qt5 fix after Qt5.3 and Qt5.4 are no longer used.
     //       Fixed in Qt5.5 and above: https://bugreports.qt.io/browse/QTBUG-43473
-#if QT_VERSION >= 0x050000
+#if QT_VERSION <= 0x050500
```

Comment:
Should probably be `<` instead of `<=` to match the mentioned/linked resolution of Qt 5.5.